src/components/chat/ChatMessages.tsx

```
@@ -0,0 +1,442 @@
+  return (
+    <Button
+      variant="outline"
+      size="sm"
+      className={`h-10 w-10 rounded-full transition-all duration-150 ease-out ${
+        !isAtBottom
+          ? 'translate-y-0 scale-100 opacity-100'
+          : 'pointer-events-none translate-y-4 scale-95 opacity-0'
+      }`}
+      onClick={scrollToBottom}
+    >
+      <svg className="h-5 w-5" fill="none" stroke="currentColor" viewBox="0 0 24 24">
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)